android/cli/commands/_build.js

```diff
@@ -3697,6 +3697,17 @@ AndroidBuilder.prototype.generateAndroidManifest = function generateAndroidManif
 		});
 	}
 
+	if (this.realTargetSDK >= 24) {
+		Object.keys(finalAndroidManifest.application.activity).forEach(function (name) {
+			var activity = finalAndroidManifest.application.activity[name];
+			if (!activity.configChanges) {
+				activity.configChanges = ['density'];
```

Comment:
Is `density` the only possible value in `configChanges`? Otherwise it would break other configs. And we should warn them to put it there if they don't do so far.